artifacts/centosstream/centos-stream_test.go

```diff
@@ -70,9 +69,9 @@ var _ = Describe("CentosStream", func() {
 				ExampleUserData: docs.UserData{
 					Username: "cloud-user",
 				},
-				AdditionalLabels: map[string]string{
-					instancetype.DefaultInstancetypeLabel: "u1.small",
-					instancetype.DefaultPreferenceLabel:   "centos.stream9",
```

Comment:
Same here, did you change it to `stream8`?

Reply:
Copy paste error, thanks!